### DISCUSSION TITLE
OrientedRepPoints config with `ms` in MMRotate seems not to use `ms` RResize transformation

### DISCUSSION THREAD

**Discussion by songyuc** [open-mmlab/mmrotate#671] [Q&A] [answered]
Hi, guys,
I am learning the OrientedRepPoints model in MMRotate. But I find the `oriented_reppoints_r50_fpn_40e_dota_ms_le135.py` config seems not to use `ms` RResize transformation, when training with command of `python tools/misc/browse_dataset.py configs/oriented_reppoints/oriented_reppoints_r50_fpn_40e_dota_ms_le135.py`. I debugged it in PyCharm, and find its pipeline is shown like:
![image](https://user-images.githubusercontent.com/27288110/209649783-944801c8-5777-432a-9299-e67da0ac7cca.png)
which shows it uses the `img_scale=(1024, 1024)` RResize, instead of the `img_scale=[(1333, 768), (1333, 1280)]` RResize.

Could you please offer some guide?
Your answer and explanation will be appreciated!

**Comment by yangxue0827** [comment, marked as answer]
`MS` in MMRotate is offline, you should use [`MS` config](https://github.com/open-mmlab/mmrotate/blob/main/tools/data/dota/split/split_configs/ms_trainval.json) when preparing datasets (crop DOTA dataset).

**Reply by songyuc** [reply]
Hi, @yangxue0827,
I have used [[split dota dataset]](https://github.com/open-mmlab/mmrotate/blob/main/tools/data/dota/README.md#split-dota-dataset) in MMRotate, however, which does not require a model config as input.
So, what do you mean with "use [MS config](https://github.com/open-mmlab/mmrotate/blob/main/tools/data/dota/split/split_configs/ms_trainval.json) when preparing datasets (crop DOTA dataset)"?

**Reply by yangxue0827** [reply]
`ms` means multi-scale crop. It is different from the common multi-scale online training (e.g. img_scale=[(1333, 768), (1333, 1280)]), but the purpose is the same. 
<img width="913" alt="image" src="https://user-images.githubusercontent.com/29257168/211769238-c43bde3d-c1c5-439c-a01d-54004bb9d6a1.png">

**Reply by songyuc** [reply]
Thanks sincerely for your explanation!
